freeipa/src/test/java/com/sequenceiq/freeipa/service/stack/FreeipaModifyProxyConfigServiceTest.java

```diff
@@ -129,14 +129,14 @@ void failToStartFlow() {
         OperationStatus failedOperationStatus = mock(OperationStatus.class);
         when(operationConverter.convert(failedOperation)).thenReturn(failedOperationStatus);
 
-        OperationStatus result = underTest.modifyProxyConfig(ENV_CRN, ACCOUNT_ID);
+        OperationStatus result = underTest.modifyProxyConfig(ENV_CRN, null, ACCOUNT_ID);
 
         assertThat(result).isEqualTo(failedOperationStatus);
     }
 
     @Test
     void success() {
-        OperationStatus result = underTest.modifyProxyConfig(ENV_CRN, ACCOUNT_ID);
+        OperationStatus result = underTest.modifyProxyConfig(ENV_CRN, null, ACCOUNT_ID);
```

Comment:
Is the test code path the same if the old proxy crn is provided?

Reply:
adding/removing a proxy is a valid use case too, and at this point there is no difference if any of them are null